src/types.ts

```diff
@@ -1,3 +1,64 @@
+import { render } from "@testing-library/react";
+import { mount } from "enzyme";
+
+// This is just a helpful rename of the interface so we can read the below types more easily
+export type FullProps<C extends React.ComponentType> = React.ComponentProps<C>;
+
+export type RenderEnzyme<
+  Component extends React.ComponentType,
+  Props extends Partial<FullProps<Component>>
+> = (
+  // By using the spread operator in this type, we leverage the optionality of the entire argument itself.
+  // IE: If the caller needs no more required props, we don't require they provide test props. But if
```

Comment:
nit; skimming or searching through the comments, one might mistake this for a browser reference. maybe just use eg of ex

Reply:
Making sure - this is the return type for renderWrapper(), correct?

Reply:
Absolutely. (And will update the abbreviation)